package.json

```
@@ -5,9 +5,11 @@
   "scripts": {
     "generate": "node generate-routes.js",
     "postgenerate": "make format",
-    "format": "prettier --write --ignore-path .gitignore ."
+    "format": "prettier --write --ignore-path .gitignore .",
+    "start:fake-seam-connect": "fake-seam-connect --seed"
   },
   "devDependencies": {
+    "@seamapi/fake-seam-connect": "^1.65.5",
```

Let's pin this one as well. It will make reproducible CI easier since we won't need to check the lockfile to see what version was used. 

```suggestion
    "@seamapi/fake-seam-connect": "1.65.5",
```

---

863b945967a9bda4b6189df79686e6f0f8fc2668